Formula/fceux.rb

```diff
@@ -1,43 +1,30 @@
 class Fceux < Formula
   desc "All-in-one NES/Famicom Emulator"
   homepage "https://fceux.com/"
-  url "https://downloads.sourceforge.net/project/fceultra/Source%20Code/2.2.3%20src/fceux-2.2.3.src.tar.gz"
-  sha256 "4be6dda9a347f941809a3c4a90d21815b502384adfdd596adaa7b2daf088823e"
-  revision 4
+  # version "2.3.0"
+  license "GPL-2.0-only"
+  revision 1
 
-  bottle do
-    cellar :any
-    sha256 "7c7550b97011321d5d48f8f689c7158223aee5054698a6c707a185404e469e35" => :catalina
-    sha256 "800e46a45f554876ad2a63ea6a62f6d672e5aefd2c9cca8f58fe615b82eb9ea7" => :mojave
-    sha256 "3f587de213706a92fb02b14676514f6cba079e3c3b7ded2e57a8e718ebf9cf20" => :high_sierra
+  stable do
+    url "https://downloads.sourceforge.net/project/fceultra/Binaries/2.3.0/fceux-2.3.0.tar.gz"
+    sha256 "f166f2fea084162f5b823bbbc6843141e23908ae4ee69c8c4e0849b320d7a1dc"
   end
 
-  # Does not build: some build scripts rely on Python 2 syntax
-  disable! date: "2020-06-30", because: :does_not_build
+  head do
+    url "https://github.com/TASVideos/fceux.git"
+  end
```

Comment:
```suggestion
  head "https://github.com/TASVideos/fceux.git"
```
This doesn't need to be a block. You need to move this to just after the `license` line.